sdk/dotnet/S3/GetBucketObject.cs

```diff
@@ -95,12 +95,6 @@ public sealed class GetBucketObjectArgs : Pulumi.InvokeArgs
         [Input("bucket", required: true)]
         public string Bucket { get; set; } = null!;
 
-        /// <summary>
-        /// (Optional) Whether or not to use [Amazon S3 Bucket Keys](https://docs.aws.amazon.com/AmazonS3/latest/dev/bucket-key.html) for SSE-KMS.
-        /// </summary>
-        [Input("bucketKeyEnabled")]
-        public bool? BucketKeyEnabled { get; set; }
```

Comment:
Breaking?

Reply:
nah, it's a data source so should be computed only - that's a cleanup item it seems...